internal/eea/entities.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2024 Stacklok, Inc
```

Comment:
Previously part of the helpers package. Moved under eea, because only the eea code uses these functions.